xrally_kubernetes/service.py

```diff
@@ -78,3 +154,75 @@ def list_namespaces(self):
                  "uid": r.metadata.uid,
                  "labels": r.metadata.labels}
                 for r in self.v1_client.list_namespace().items]
+
+    @atomic.action_timer("kube.create_namespace")
+    def create_namespace(self, name):
+        """Create namespace.
+
+        :param name: namespace name
+        """
+        manifest = {
+            "apiVersion": "v1",
+            "kind": "Namespace",
+            "metadata": {
+                "name": name,
+                "labels": {
+                    "role": "test-%s" % name
+                }
+            }
+        }
+        self.v1_client.create_namespace(body=manifest)
+        LOG.info("Namespace %s create started" % name)
+
+    @atomic.action_timer("create_namespace_and_wait_active_status")
+    def create_namespace_and_wait_active(self, name, sleep_time=5,
```

Comment:
1) I do not see any reasons to have a separate method with waiting. Let's use "wait_for_acrive" argument at the original `create_namespace` method.

2) The numbers of retries and sleep are something which is common for the whole cluster. Let's use config options for these.  

  https://github.com/openstack/rally-openstack/blob/master/rally_openstack/cfg/nova.py#L19-L32
  Also, the special list_opts method should be created to make Rally register new options - https://github.com/openstack/rally-openstack/blob/master/rally_openstack/cfg/opts.py#L43 . + this should be un-commented https://github.com/xrally/xrally-kubernetes/blob/master/setup.py#L49